src/mainboard/starfive/visionfive2/bt0/src/ddr_start.rs

```diff
@@ -2,7 +2,7 @@ use crate::ddrlib::{MemCfg, MemSet};
 use crate::init::{self, read32, write32};
 
 // TODO: define build time parameters (!)
-const VAL_X1: u32 = if cfg!(dram_size = "8G") || cfg!(dram_size = "4G") {
+const VAL_X1: u32 = if cfg!(dram_size = "8G") || cfg!(dram_size = "4G") || true {
```

Comment:
Is this temporary? These will always evaluate to `true`.

Reply:
Yea. It's one commit hardcoding a 4GB config (what I have) for now. The cfg somehow didn't make it into the build.

The DRAM init is not working yet, missing some clocks maybe - I'll check today.

Reply:
I've rework this to a constant. If you can figure out why the `cfg!()` stuff doesn't work, please file a PR.

Eventually, the DRAM size would need to be read from an EEPROM.